testing/trino-product-tests/src/main/java/io/trino/tests/product/deltalake/TestDeltaLakeColumnMappingMode.java

```diff
@@ -67,35 +67,245 @@ public void testColumnMappingModeName()
+            }
+            onDelta().executeQuery("INSERT INTO default." + tableName + " VALUES (null)");
+
+            assertThat(onTrino().executeQuery("SHOW STATS FOR delta.default." + tableName))
+                    .containsOnly(ImmutableList.of(
+                            row("a_number", null, null, 0.09090909091, null, "0", "9"),
+                            row(null, null, null, null, 11.0, null, null)));
+        }
+        finally {
+            onTrino().executeQuery("DROP TABLE delta.default." + tableName);
+        }
+    }
+
+    @Test(groups = {DELTA_LAKE_DATABRICKS, DELTA_LAKE_OSS, PROFILE_SPECIFIC_TESTS})
+    public void testShowStatsOnPartitionedForColumnMappingModeName()
+    {
+        String tableName = "test_dl_show_stats_partitioned_for_column_mapping_mode_" + randomTableSuffix();
+
+        onDelta().executeQuery("" +
+                "CREATE TABLE default." + tableName +
+                " (a_number INT, part INT)" +
+                " USING delta " +
+                " PARTITIONED BY (part) " +
+                " LOCATION 's3://" + bucketName + "/databricks-compatibility-test-" + tableName + "'" +
+                " TBLPROPERTIES (" +
+                " 'delta.columnMapping.mode' = 'name'," +
+                " 'delta.minReaderVersion' = '2'," +
+                " 'delta.minWriterVersion' = '5')");
+
+        try {
+            for (int value = 0; value < 10; value++) {
+                onDelta().executeQuery("INSERT INTO default." + tableName + " VALUES (" + -1 * value + ", '" + value + "')");
+            }
+            onDelta().executeQuery("INSERT INTO default." + tableName + " VALUES (null, null)");
+
+            assertThat(onTrino().executeQuery("SHOW STATS FOR delta.default." + tableName))
+                    .containsOnly(ImmutableList.of(
+                            row("a_number", null, null, 0.09090909091, null, "-9", "0"),
+                            row("part", null, 10.00000000000, 0.09090909091, null, null, null),
+                            row(null, null, null, null, 11.0, null, null)));
+        }
+        finally {
+            onTrino().executeQuery("DROP TABLE delta.default." + tableName);
+        }
+    }
+
+    @Test(groups = {DELTA_LAKE_DATABRICKS, DELTA_LAKE_OSS, PROFILE_SPECIFIC_TESTS})
+    public void testUnsupportedOperationsColumnMappingModeName()
+    {
+        String tableName = "test_dl_column_mapping_mode_" + randomTableSuffix();
+
+        onDelta().executeQuery("" +
+                "CREATE TABLE default." + tableName +
+                " (a_number INT, a_string STRING)" +
+                " USING delta " +
+                " LOCATION 's3://" + bucketName + "/databricks-compatibility-test-" + tableName + "'" +
+                " TBLPROPERTIES (" +
                 " 'delta.columnMapping.mode'='name'," +
                 " 'delta.minReaderVersion'='2'," +
                 " 'delta.minWriterVersion'='5')");
```

Comment:
What happens if you create table with `'delta.columnMapping.mode'='name'`, but omit reader/writer version explicit settings? will Delta/Databricks infer them to be 2 and 5 respectively?

Reply:
It will be inferred automatically. Removed `minReaderVersion` & `minWriterVersion` including other places.

Reply:
> It will be inferred automatically.

awesome.

we need to test with defaults because this is what users will do

> Removed minReaderVersion & minWriterVersion including other places.

thanks!